File: packages/android/app/src/main/java/com/byoky/app/ui/theme/Theme.kt
package com.byoky.app.ui.theme

import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.lightColorScheme
import androidx.compose.runtime.Composable
import androidx.compose.ui.graphics.Color

// Brand palette (matches landing page)
val Accent = Color(0xFFFF4F00)
val AccentHover = Color(0xFFFF6A2A)
val AccentSoft = Color(0x1FFF4F00)
val BgMain = Color(0xFFF5F5F4)
val BgRaised = Color(0xFFF5F5F4)
val BgCard = Color(0xFFFFFFFF)
val BgHover = Color(0xFFE7E5E4)
val TextPrimary = Color(0xFF1C1917)
val TextSecondary = Color(0xFF57534E)
val TextMuted = Color(0xFFA8A29E)
val Danger = Color(0xFFDC2626)
val Success = Color(0xFF16A34A)
val Warning = Color(0xFFD97706)
val Border = Color(0xFFE7E5E4)

private val ByokyColorScheme = lightColorScheme(
    primary = Accent,
    onPrimary = Color.White,
    primaryContainer = AccentSoft,
    onPrimaryContainer = Accent,
    secondary = AccentHover,
    background = BgMain,
    onBackground = TextPrimary,
    surface = BgCard,
    onSurface = TextPrimary,
    surfaceVariant = BgRaised,
    onSurfaceVariant = TextSecondary,
    error = Danger,
    onError = Color.White,
    outline = Border,
)

@Composable
fun ByokyTheme(content: @Composable () -> Unit) {
    MaterialTheme(
        colorScheme = ByokyColorScheme,
        content = content,
    )
}

File: packages/ios/Byoky/Extensions/Theme.swift
import SwiftUI

enum Theme {
    // Brand palette: --accent: #FF4F00
    static let accent = Color("Accent")
    static let accentSoft = Color("AccentSoft")
    static let accentHover = Color("AccentHover")

    // Light palette: --bg: #fafaf9, --bg-raised: #f5f5f4, --bg-card: #ffffff
    static let bgMain = Color("BgMain")
    static let bgRaised = Color("BgRaised")
    static let bgCard = Color("BgCard")

    // Semantic
    static let danger = Color(red: 0.863, green: 0.149, blue: 0.149) // #dc2626
    static let success = Color(red: 0.086, green: 0.639, blue: 0.290) // #16a34a

    // Text (warm stone)
    static let textPrimary = Color(red: 0.110, green: 0.098, blue: 0.090) // #1c1917
    static let textSecondary = Color(red: 0.341, green: 0.325, blue: 0.306) // #57534e
    static let textMuted = Color(red: 0.659, green: 0.635, blue: 0.620) // #a8a29e

    // Borders
    static let border = Color(red: 0.906, green: 0.898, blue: 0.886) // #e7e5e4
    static let borderHover = Color(red: 0.839, green: 0.827, blue: 0.816) // #d6d3d1
}
